src/ui/public/courier/search_source/search_source.js

```diff
@@ -267,43 +275,28 @@ export function SearchSourceProvider(Promise, Private, config) {
      * Get the parent of this SearchSource
      * @return {undefined|searchSource}
      */
-    getParent() {
+    getParent = () => {
```

Comment:
I'm sorry, but I really don't like needless function arrows in class bodies. They're useful when you need to bind a handler to the `this` context, but using them everywhere is unnecessary.